extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -159,27 +159,32 @@ export function getElement(ampdoc, selector, el, selectionMethod) {
     return null;
   }
 
+  const elWin = el.ownerDocument.defaultView;
   // Special case for root selector.
   if (selector == ':host' || selector == ':root') {
-    const elWin = ampdoc.win;
     const parentEl = elWin.frameElement && elWin.frameElement.parentElement;
```

Comment:
Please switch to use `getParentWindowFrameElement` from `service.js` for frameElement calculation. Looking at this, I actually no idea why we need `.parentElement` here. I don't think it's needed. The `.frameElement` (returned by `getParentWindowFrameElement`) is in itself the `parentEl`.